pkgs/by-name/ha/hamburger/package.nix

```diff
@@ -0,0 +1,178 @@
+{
+  stdenv,
+  lib,
+  fetchFromGitHub,
+  fetchpatch,
+  cmake,
+  pkg-config,
+  writableTmpDirAsHomeHook,
+  juce,
+  libx11,
+  libxrandr,
+  libxinerama,
+  libxext,
+  libxcursor,
+  libxtst,
+  freetype,
+  fontconfig,
+  webkitgtk_4_1,
+  curl,
+  alsa-lib,
+  libsysprof-capture,
+  pcre2,
+  util-linux,
+  libselinux,
+  libsepol,
+  libthai,
+  libdatrie,
+  libxdmcp,
+  libdeflate,
+  lerc,
+  xz,
+  libwebp,
+  libxkbcommon,
+  libepoxy,
+  sqlite,
+  nix-update-script,
+
+  buildVST3 ? true,
+  buildStandalone ? true,
+  buildLV2 ? true,
+  buildCLAP ? true,
+}:
+let
+  clap-juce-extensions = fetchFromGitHub {
+    owner = "free-audio";
+    repo = "clap-juce-extensions";
+    rev = "645ed2fd0949d36639e3d63333f26136df6df769";
+    hash = "sha256-Lx88nyEFjPLA5yh8rrqBdyZIxe/j0FgIHoyKcbjuuI4=";
+    fetchSubmodules = true;
+  };
+
+  formats = lib.concatStringsSep " " [
+    (lib.optionalString buildVST3 "VST3")
+    (lib.optionalString buildStandalone "Standalone")
+    (lib.optionalString buildLV2 "LV2")
+  ];
+in
+stdenv.mkDerivation (finalAttrs: {
+  pname = "hamburger";
+  version = "0.5";
+
+  src = fetchFromGitHub {
+    owner = "Davit-G";
+    repo = "Hamburger";
+    rev = "2ce592c165b00aaf42edaa0af7aed0cbe7df27fe";
```

Comment:
nit: 
```suggestion
  tag = "v${finalAttrs.version}";
 ```

Reply:
Oops, forgot to change the version before submiting, thanks for pointing this out. I believe that current unstable has a bunch of cool features/fixes, let's package it.